include/oneapi/dpl/internal/exclusive_scan_by_segment_impl.h

```diff
@@ -127,7 +127,7 @@ exclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
         flags[0] = 1;
     }
 
-    transform(::std::forward<Policy>(policy), first1, last1 - 1, first1 + 1, _flags.get() + 1,
```

Comment:
I think this is correct, but I also think we can add forwarding to the `policy2` creation @ line 154 below.

Reply:
Done.